transactions/buy_item.cdc

```diff
@@ -1,19 +1,20 @@
-import FungibleToken from "../contracts/FungibleToken.cdc"
-import NonFungibleToken from "../contracts/NonFungibleToken.cdc"
-import FlowToken from "../contracts/FlowToken.cdc"
-import ExampleNFT from "../contracts/ExampleNFT.cdc"
-import NFTStorefront from "../contracts/NFTStorefront.cdc"
+import FlowToken from 0x0ae53cb6e3f42a79
+import FungibleToken from "../contracts/utility/FungibleToken.cdc"
+import NonFungibleToken from "../contracts/utility/NonFungibleToken.cdc"
+import ExampleNFT from "../contracts/utility/ExampleNFT.cdc"
+import NFTStorefrontV2 from "../contracts/NFTStorefrontV2.cdc"
 
-transaction(listingResourceID: UInt64, storefrontAddress: Address) {
+transaction(listingResourceID: UInt64, storefrontAddress: Address, commissionRecipient: Address) {
```

Comment:
again here, i'd probably imagine the commissionRecipient would normally be hard-coded into the tx - rather than subject to user input.

Reply:
Again want to make flexible as much as possible, It is up to the dApp developers to change